packages/angular/projects/ui-angular/src/lib/primitives-poc/text-field/text-field-component/text-field.component.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChild,
+  Input,
+  SimpleChanges,
+} from '@angular/core';
+import { InputDirective } from '../text-field-directives/input.directive';
+
+const $COMPONENT_SELECTOR: string = 'amplify-poc-text-field';
+@Component({
+  selector: $COMPONENT_SELECTOR,
+  templateUrl: 'text-field.component.html',
+})
+export class AmplifyTextFieldsComponent implements AfterContentInit {
+  @Input() describedBy: string;
+  @Input() labelHidden: boolean = false;
+  @Input() hasError: boolean = false;
+
+  @ContentChild(InputDirective) childInput: InputDirective | undefined;
```

Comment:
```suggestion
  @ContentChild(InputDirective) childInput: InputDirective;
```

we have [`strictNullChecks`](https://www.typescriptlang.org/tsconfig#strictNullChecks) off (by default), so we don't need to write `undefined`.